Changelog.md

```diff
@@ -1,6 +1,9 @@
 # CHANGELOG
 All notable changes to this project will be documented here.
 
+## [unreleased]
```

Comment:
Should this now be v2.1.2?

Reply:
Yep, you're correct. I've updated it now.